Sources/SpeziOnboarding/ConsentView.swift

```diff
@@ -197,6 +228,132 @@ public struct ConsentView<ContentView: View, Action: View>: View {
     }
 }
 
+
+/// Extension of ``ConsentView`` enabling the export of the signed consent page in the onboarding flow.
+extension ConsentView {
+    /// Creates a view representation of the consent content, ready for PDF export via SwiftUIs `ImageRenderer`.
+    /// At the moment, this is
+    ///
+    /// This function constructs a view for presenting the markdown consent form. It combines the
+    /// given markdown and the user's signature with details such as the date of export. It can be
+    /// used to create exportable PDF documents of the consent form.
+    ///
+    /// - Parameters:
+    ///   - markdown: The markdown consent content as an `AttributedString`.
+    ///
+    /// - Returns: A SwiftUI `View` representation of the consent content and signature.
+    ///
+    /// - Note: This function avoids the use of asynchronous operations.
+    /// Asynchronous tasks are incompatible with SwiftUI's `ImageRenderer`,
+    /// which expects all rendering processes to be synchronous.
+    func exportBody(markdown: AttributedString) -> some View {
+        VStack {
+            HStack {
+                Spacer()
+                
+                Text("Exported: \(DateFormatter.localizedString(from: Date(), dateStyle: .medium, timeStyle: .short))")
+                    .font(.caption)
+            }
+            .padding()
+            
+            Text("Spezi Consent")
+                .font(.title)
+            
+            Text(markdown)
+                .padding()
+            
+            Spacer()
+            
+            ZStack(alignment: .bottomLeading) {
+                Rectangle()
+                    .fill(.secondary)
+                    .frame(maxWidth: .infinity, maxHeight: 1)
+                    .padding(.horizontal, 20)
+                    .padding(.bottom, 30)
+                Text("X")
+                    .font(.title2)
+                    .foregroundColor(.secondary)
+                    .padding(.horizontal, 20)
+                    .padding(.bottom, 30 + 2)
+                Text(name.formatted(.name(style: .long)))
+                    .font(.subheadline)
+                    .foregroundColor(.secondary)
+                    .padding(.horizontal, 20)
+                    .padding(.bottom, 30 - 18)
+                Image(uiImage: signature.image(
+                    from: .init(x: 0, y: 0, width: signatureSize.width, height: signatureSize.height),
+                    scale: UIScreen.main.scale
+                ))
+            }
+            .frame(width: signatureSize.width, height: signatureSize.height)
```

Comment:
Could we reuse the signature view here and just display the image on top of the view so we don't have to re-implement, e.g., the "X" and everything else?

Reply:
I thought that too but sadly this doesn't work out of the box as the `ImageRenderer` is quite picky (I also highlighted that in the DocC docs). However, what we may be able to do is provide different versions of the `SignatureView` via a flag that indicates export (hiding/showing specific parts of the view). Not sure if that is the best/cleanest solution, but this would at least avoid parts of the code duplication. Will give it a try.

Reply:
Thank you for the context. As it only works for SwiftUI views we can not use the `PKCanvas` here but we could e.g. separate the `SignatureView` in a Background (that is pure SwiftUI and can be reused) and the background + canvas that builds out the main `SignatureView`